project/CONTRIBUTING.md

```diff
@@ -324,6 +324,9 @@ but some extra rules must be respected here:
   - PHP versions that are under the [green zone][php_supported_versions] (actively maintained) **MUST NO** be dropped, even on master.
   - If it's a Symfony package, at least the last LTS version **MUST** be supported, even on master.
   - Generally, don't drop dependency version it it doesn't have a big impact on the code.
+  - Backward Compatible code related to the dropped version **MUST** be dropped on the same PR.
+    This will permit to see if this version dropping **really worth it** or not.
```

Comment:
*is really worth it?

Reply:
"It's really worth it! Go!"

Indeed. See? Your English is not that bad! :stuck_out_tongue: